mix.exs

```diff
@@ -25,9 +25,9 @@ defmodule Brook.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:brook_serializer, "~> 1.0"},
+      {:brook_serializer, git: "https://github.com/jeffgrunewald/brook_serializer.git", branch: "change_api"},
```

Comment:
Did you mean to do this?

Reply:
I did, only because the brook serializer wasn’t merged/published when I pushed this up. I mentioned in person this should be changed before merging the rest of the pr

Reply:
In fact it looks like it still hasn’t been published yet so this is technically required to make the tests pass